docs/create-a-neuron.md

```diff
@@ -37,12 +39,15 @@ dfx --identity neuron-hotkey identity get-principal
 ```
 Example output: `wuyst-x5tpn-g5wri-mp3ps-vjtba-de3xs-w5xgb-crvek-tucbe-o5rqi-mae`
 
-**Note:** This hotkey is used for NNS proposal submissions only.
+## (Optional) Hardware wallet identity
+
+In the NNS UI you can also add a hardware wallet as a controlling entity. You will not be able to submit proposals with the hardware wallet entity though. But you can keep all your funds on the hardware wallet.
 
 ### Create and Manage Neuron via NNS Frontend Dapp and Internet Identity
 
-1. Send at least 11 ICPs to your hardware wallet address.
-2. Navigate to the Neurons tab and create a Neuron by staking at least 10 ICP from your hardware wallet. Confirm the transaction on your hardware wallet.
-3. After the Neuron is created, confirm to "Add NNS Dapp as hotkey" in the dialogue and on your hardware wallet.
-4. Set the dissolve delay to at least 6 months and confirm the choice.
-5. Copy the Neuron ID from the Web UI, for use in the next steps.
+1. Send at least 11 ICPs (10 of which are required for proposals submisison, and 1 is aiming to cover all potential fees) to your NNS UI address OR the hardware wallet address.
+3. Navigate to the Neurons tab in the NNS UI and create a Neuron by staking at least 10 ICP. Confirm the transaction on your hardware wallet.
```

Comment:
```suggestion
Confirm the transaction on your hardware wallet.
```
This is only needed if the neuron is controlled through a hardware wallet.

Reply:
Agreed, cleaning up now. PTAL

Reply:
Looks good to me :)